projects/components/src/datagrid/filters/datagrid-multiselect-filter.component.ts

```diff
@@ -127,6 +127,10 @@ export class DatagridMultiSelectFilterComponent extends DatagridFilter<string[],
             !!Object.keys(this.formGroup.getRawValue()).filter((frmCtrl) => this.formGroup.get(frmCtrl).value).length
         );
     }
+
+    protected setFocus() {
```

Comment:
I think a better approach is to have a method called `getFocusableElement()`, then the callers just have to return the element instead of every implementation calling `focus()`.

Reply:
I like the `focus()` since it seems a more complete approach to me.
If I come accross to `getFocusableElement()` function - it is not that clear how the returned element will be used. Yes, the name suggests to focus, but yet - not that clean. In addition, I'm not sure if there is a case when something else may be needed when focusing the element. On the other hand `focus()` is more concise and clear.

I agree that the very `.focus()` call can be reused in the base class but I don't see it has the advantages as the other approach.